compiler/src/iree/compiler/Pipelines/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ iree_cc_library(
     MLIRIR
     MLIRPass
     MLIRSupport
+    StablehloPasses
```

Comment:
No deps on input dialects outside of their plugins

Reply:
fixed.